src/charm.py

```diff
@@ -125,23 +116,30 @@ def _configure_sdcore_smf(
             self.unit.status = WaitingStatus("Waiting for container to be ready")
             event.defer()
```

Comment:
No need for defer here since pebble_ready calls this method

Reply:
Removed.